src/stage1/parse_f128.c

```diff
@@ -10,6 +10,22 @@
 #include <string.h>
 #include <math.h>
 
+// Every OSes seem to define endianness macros in different files.
+#if defined(__APPLE__)
+  #include <machine/endian.h>
+  #define __BIG_ENDIAN BIG_ENDIAN
```

Comment:
Should avoid defining any `__` prefixed defines:
> Each name that contains a double underscore (__) or begins with an underscore followed by an uppercase letter is reserved to the implementation for any use.

Instead, define something like `ZIG_ENDIAN` on all platforms and use that.